eng/pipelines/templates/steps/vmr-compare.yml

```
@@ -178,18 +162,24 @@ steps:
         -assetFilter $assetFilter `
         -nonShipping:$nonShipping
 
+# If running on MacOS, the total output size is too large
+# to download entirely. Instead, we use the patterns of the artifacts
+# that should validated on Mac.
 - template: ../steps/vmr-download-artifact.yml
   parameters:
     displayName: Download VMR Artifacts
     buildId: $(GetBuildInfo.UnifiedBuildRunId)
-    ${{ if eq(parameters.assetType, 'Blob') }}:
+    ${{ if eq(parameters.OS, 'Darwin') }}:
       itemPattern: |
-        *_Artifacts/assets/**
-        *_Artifacts/manifests/**
-    ${{ elseif eq(parameters.assetType, 'Package') }}:
-      itemPattern: |
-        *_Artifacts/packages/**
-        *_Artifacts/manifests/**
+        iOS*_Artifacts/assets/**
+        iOS*_Artifacts/packages/**
+        iOS*_Artifacts/manifests/**
```

tvOS artifacts should be included as well if we want to cover all Apple platform artifacts.

```suggestion
        iOS*_Artifacts/assets/**
        iOS*_Artifacts/packages/**
        iOS*_Artifacts/manifests/**
        tvOS*_Artifacts/assets/**
        tvOS*_Artifacts/packages/**
        tvOS*_Artifacts/manifests/**

```